pipeline/transform.py

```diff
@@ -7,10 +7,7 @@
 
 PAGER_ALERT_LEVELS = ['green', 'yellow', 'orange', 'red']
 READING_STATUS = ['automatic', 'reviewed', 'deleted']
-NETWORKS = ['ak', 'at', 'ci', 'hv', 'ld', 'mb', 'nc', 'nm',
-            'nn', 'pr', 'pt', 'se', 'us', 'uu', 'uw']
-MAGNITUDE_TYPES = ['md', 'ml', 'ms', 'mw', 'me', 'mi', 'mb', 'mlg']
-EARTHQUAKE_TYPES = ['earthquake', 'quarry']
+NETWORK_NAME_LENGTH = 2
```

Comment:
Do we know for a fact the network will only have a length of two? How can we be sure of this?

Reply:
The networks all follow a code set by this organisation. If they change their naming scheme then we can simply amend the variable but I don't think they change it often

 https://www.fdsn.org/networks/?permanent=1&page=1

Reply:
I looked up the various networks the API uses and they all seem to have a length of 2 https://earthquake.usgs.gov/monitoring/operations/network.php